### ISSUE TITLE
Module docstrings have inconsistent behaviour on hover

### ISSUE THREAD

**Issue by thomasaarholt** [microsoft/pylance-release#3636]
Type: <b>Bug</b>

VScode has excellent support for showing docstrings, also of modules.
# Behaviour
The following video shows an inconsistency for when docstrings are shown. Here I am showing two packages that are well-known, but this is much more important when the packages are obscure and one wants to learn more.

In the video, docstrings are shown for all modules except towards the end, where they are shown as just `(module) pyplot`. I believe it would be beneficial to the community if the full docstring was shown in all of these cases.

https://user-images.githubusercontent.com/2721423/202221257-f8a204db-ee3c-4197-9b1a-bea665b1546e.mov

## Expected vs. Actual

I expected the docstrings to show in all cases.

## Steps to reproduce:
Copy the following into python environment that has numpy and matplotlib installed, then hover on top of the module and function names.
```python
import numpy

from numpy import arange # importing function from module

from matplotlib import pyplot # importing module from module

import matplotlib.pyplot

import matplotlib.pyplot as plt


# Below are just to silence any other warnings
numpy
arange
matplotlib.pyplot
pyplot
plt
```

# Diagnostic data

-   Python version (& distribution if applicable, e.g. Anaconda): 3.10.8
-   Type of virtual environment used (e.g. conda, venv, virtualenv, etc.): Global
-   Value of the `python.languageServer` setting: Default

<details>

<summary>Output for <code>Python</code> in the <code>Output</code> panel (<code>View</code>→<code>Output</code>, change the drop-down the upper-right of the <code>Output</code> panel to <code>Python</code>)
</summary>

<p>

```
LSP Notebooks experiment is enabled
LSP Notebooks interactive window support is enabled
Python interpreter path: /usr/local/bin/python
> conda info --json
> /usr/local/bin/python ~/.vscode-server/extensions/ms-python.python-2022.18.2/pythonFiles/get_output_via_markers.py ~/.vscode-server/extensions/ms-python.python-2022.18.2/pythonFiles/interpreterInfo.py
Starting Pylance language server.
> /usr/local/bin/python -m black --line-length 88 --experimental-string-processing --diff --quiet ./adivino/utils/utils.py
cwd: .
Jupyter provided interpreter path override: /usr/local/bin/python
Jupyter provided interpreter path override: /usr/local/bin/python
Jupyter provided interpreter path override: /usr/local/bin/python
Jupyter provided interpreter path override: /usr/local/bin/python
Jupyter provided interpreter path override: /usr/local/bin/python
Jupyter provided interpreter path override: /usr/local/bin/python

```

</p>
</details>

<details>

<summary>User Settings</summary>

<p>

```

languageServer: "Pylance"

formatting
• provider: "black"
• blackArgs: "<placeholder>"

```

</p>
</details>

Extension version: 2022.18.2
VS Code version: Code 1.73.1 (6261075646f055b99068d3688932416f2346dd3b, 2022-11-09T02:22:48.959Z)
OS version: Darwin arm64 22.1.0
Modes:
Sandboxed: No
Remote OS version: Linux arm64 5.15.49-linuxkit
Connection to 'attached-container+7b22636f6e7461696e65724e616d65223a222f61646976696e6f227d' could not be established

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M1 (8 x 24)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|4, 3, 3|
|Memory (System)|16.00GB (0.11GB free)|
|Process Argv|--crash-reporter-id b8a88393-8180-45bb-9724-18ccbb492bdb|
|Screen Reader|no|
|VM|0%|

|Item|Value|
|---|---|
|Remote|Container eu.gcr.io/nube-hub/fabrica/bayescast:8.0.0-5558b32b48 (bayescast)|
|OS|Linux arm64 5.15.49-linuxkit|
|CPUs|unknown (6 x 0)|
|Memory (System)|8.74GB (5.23GB free)|
|VM|0%|

Connection to 'attached-container+7b22636f6e7461696e65724e616d65223a222f61646976696e6f227d' could not be established
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscoreces:30445986
pythondataviewer:30285071
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593cf:30376535
pythonvs932:30410667
cppdebug:30492333
vsclangdc:30486549
c4g48928:30535728
dsvsc012cf:30540253
azure-dev_surveyone:30548225
pyindex848:30577860
nodejswelcome1:30587005
fc301958:30595537
3biah626:30602489
gswce1:30612156
iaj6b796:30608729
dbltrim-noruby:30604474
89544117:30607850

```

</details>

**Comment by rchiodo** [comment]
This looks like the node type is determing whether or not we find the docstring.

**Comment by rchiodo** [comment]
This issue has been fixed in prerelease version 2022.12.21, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#20221221-7-December-2022-PreRelease)
